tests/unit/test_mechaclient.py

```diff
@@ -47,6 +48,13 @@ def test_verify_rat_board_private(bot_fx):
     assert bot_fx._rat_board is None
 
 
+def test_verify_galaxy_private(bot_fx):
+    """
+    Asserts the private reference for the Galaxy is none, after instantiation.
+    """
+    assert bot_fx._galaxy is None
```

Comment:
[error] this test is unnecessary, as it tests python's behavior rather than our code specifically.

Reply:
May need to eventually look at the other 4 tests in there as well. I was simply following their example.

Reply:
Might be wise. guess we need to schedule a review at some point 😕